holidays/locale/th/LC_MESSAGES/SE.po

```diff
@@ -59,9 +64,9 @@ msgstr "วันเมย์เดย์ (วันแรงงาน)"
 msgid "Kristi himmelsfärdsdag"
 msgstr "วันสมโภชพระเยซูเจ้าเสด็จขึ้นสวรรค์"
 
-#. National Day of Sweden.
-msgid "Sveriges nationaldag"
-msgstr "วันชาติสวีเดน"
+#. National Day.
+msgid "Nationaldagen"
+msgstr "วันชาติ"
```

Comment:
I've missed this change somehow in the earlier review last night, all current `th` l10n for non-Thailand national day naming currently includes the country name in the text at the moment (also used by [Embassy of Sweden in Bangkok's translation](https://www.facebook.com/SwedeninBangkok/posts/-glad-nationaldag-happy-swedish-national-day-english-belowในวันนี้-วันที่-6-มิถุ/1137240661768571/) ) - this is the last one 🙏 
```suggestion
msgstr "วันชาติสวีเดน"
```